applications/games_menu/modules/utils.c

```diff
@@ -198,8 +198,6 @@ bool IsGdiOptimized(const char *full_path_game)
 
 		strncpy(path, full_path_game, strlen(full_path_game) - (strlen(game) + 1));
 		snprintf(result, NAME_MAX, "%s/track01.iso", path);
```

Comment:
Better check track03.iso because user can delete two first tracks to save storage space. It's not used in most games, only few games check first track.

Reply:
Good point, do you have anything else?

Reply:
to turn on my MacWindows

Reply:
Only this.

Reply:
I already loaded the change.